app/src/main/res/layout-land/activity_main.xml

```diff
@@ -1,61 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/question_constraint_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
-    android:id="@+id/question_linear_layout">
+    android:gravity="center">
 
     <TextView
         android:id="@+id/question_text_view"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_gravity="center_horizontal"
         android:padding="24dp"
-        tools:text="@string/question_noriaki" />
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        tools:text="@string/question_noriaki"
+        android:layout_marginBottom="100dp"/>
 
-    <LinearLayout
-        android:layout_width="wrap_content"
+    <Button
+        android:id="@+id/false_button"
+        android:layout_width="100dp"
         android:layout_height="wrap_content"
-        android:layout_gravity="center_vertical|center_horizontal">
-
-        <Button
-            android:id="@+id/true_button"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="@string/true_button" />
-
-        <Button
-            android:id="@+id/false_button"
-            android:layout_width="101dp"
-            android:layout_height="wrap_content"
-            android:text="@string/false_button" />
-
-    </LinearLayout>
+        android:text="@string/false_button"
+        app:layout_constraintEnd_toStartOf="@+id/true_button"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/question_text_view" />
 
     <Button
-        android:id="@+id/cheat_button"
-        android:layout_width="wrap_content"
+        android:id="@+id/true_button"
+        android:layout_width="100dp"
```

Comment:
`ConstraintLayout` is powerful, because it allows for creating screen size agnostic layouts. By hardcoding the width here to `100dp` you are saying that this button should be 100dp wide, no matter the screen size. This in result means that on smaller screens, i.e older android phones this button can take up to 75% of the screen width, while on bigger screens (like tablet) it will take up for example 10% of the screen. See it for yourself by launching the app on the tablet  emulator.

While this is not wrong, as you may specifically want it to be that way, there are mechanisms for you to use in `ConstraintLayout` that will ensure the _ratio_ is always the same, i.e width _always_ takes up for example 50% of  the screen. Check out attributes for setting relative dimensions:

```
layout_constraintWidth_percent
layout_constraintHeigth_percent
```

and for relative positioning:

```
layout_constraintHorizontal_bias
layout_constraintVertical_bias
```

you can find further explanation [here](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout)

Reply:
Good point! Thank you for your advice. Already checked these attributes, work nice!